home-dir/homeDir.js

```diff
@@ -1,8 +1,7 @@
+import os from 'os';
 
-function homeDir(...args) {
-  return args;
+function homeDir() {
+  return os.homedir();
 }
 
-export {
-  homeDir,
-};
+export { homeDir };
```

Comment:
Lines seem to have changed for no reason. Do you think these changes could have been avoided?

Reply:
Good point. I think he must be using Prettier or some extensions.